### PR TITLE
Add libscrypt

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: docs/packages/pkg/libscrypt.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+.. _pkg.libscrypt:
+
+libscrypt
+=========
+
+-  `Official <https://github.com/technion/libscrypt>`__
+-  `Hunterized <https://github.com/hunter-packages/libscrypt>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/libscrypt/CMakeLists.txt>`__
+
+.. code-block:: cmake
```
Comment (ruslo) [inline docs/packages/pkg/libscrypt.rst]: It's better to use `literalinclude` here, see template:
* https://raw.githubusercontent.com/ruslo/hunter/b9ec25a0ffd23eeccdb692edef47f1cd03db30f9/docs/packages/pkg/foo.rst

File: examples/libscrypt/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(libscrypt-test)
+
+hunter_add_package(libscrypt)
```
Comment (ruslo) [inline examples/libscrypt/CMakeLists.txt]: Plus `DOCUMENTATION_START` / `DOCUMENTATION_END` for `literalinclude`, see:
* https://github.com/ruslo/hunter/blob/b9ec25a0ffd23eeccdb692edef47f1cd03db30f9/examples/foo/CMakeLists.txt#L12-L18

**Comment by chfast** [comment]
Testing:
https://travis-ci.org/chfast/hunter/builds/336578951
https://ci.appveyor.com/project/chfast/hunter/build/1.0.1

**Comment by chfast** [comment]
Testing:
- https://ci.appveyor.com/project/chfast/hunter/build/1.0.10
- https://travis-ci.org/chfast/hunter/builds/336619090

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3305
* https://travis-ci.org/ingenue/hunter/builds/336657915?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3306
* https://travis-ci.org/ingenue/hunter/builds/336669358?utm_source=github_status&utm_medium=notification

**Pull request merged**